types/lodash/index.d.ts

```diff
@@ -7138,6 +7174,411 @@ declare namespace _ {
         flatMap<TResult>(): LoDashExplicitArrayWrapper<TResult>;
     }
 
+    //_.flatMapDeep
+    interface LoDashStatic {
+        /**
+         * This method is like `_.flatMap` except that it recursively flattens the
+         * mapped results.
+         *
+         * @static
+         * @memberOf _
+         * @since 4.7.0
+         * @category Collection
+         * @param {Array|Object} collection The collection to iterate over.
+         * @param {Function} [iteratee=_.identity] The function invoked per iteration.
+         * @returns {Array} Returns the new flattened array.
+         * @example
+         *
+         * function duplicate(n) {
+         *   return [[[n, n]]];
+         * }
+         *
+         * _.flatMapDeep([1, 2], duplicate);
+         * // => [1, 1, 2, 2]
+         */
+        flatMapDeep<T>(
+            collection: List<Many<T>> | Dictionary<Many<T>> | NumericDictionary<Many<T>> | null | undefined
```

Comment:
I still think this should be `RecursiveArray<T>` (or possibly ListOfRecursiveArraysOrValues<T>, but I think `RecursiveArray<T>` is more accurate) instead of `Many<T>`.

Same with `flatMapDepth` - although it's not 100% accurate for `flatMapDepth`, I think it's still better than `Many<T>`.